src/components/flat-table/__snapshots__/flat-table.spec.js.snap

```diff
@@ -62,31 +62,35 @@ exports[`FlatTable when rendered with proper table data should have expected str
   z-index: 1000;
 }
 
-.c3 .c5 > div {
+.c4 .c6 > div {
   padding: 8px 24px;
 }
 
-.c1 {
+.c2 {
   border-collapse: separate;
   border-radius: 0px;
   border-spacing: 0;
   min-width: 100%;
   width: 100%;
 }
 
-.c0 {
+.c1 {
   height: 100%;
 }
 
-.c0 .c2 .c12,
-.c0 .c7,
-.c0 .c2 .c5 {
+.c1 .c3 .c13,
+.c1 .c8,
+.c1 .c3 .c6 {
   background-color: #335C6D;
   border-right: 1px solid #668592;
   color: #FFFFFF;
 }
 
-.c10 {
+.c0::-webkit-scrollbar {
```

Comment:
Shouldn't this only be added if a `Box` is used with the `scrollVariant` prop? Otherwise it should be browser default?

Reply:
Same as below. This is just snapshots